src/index.html

```diff
@@ -132,7 +131,364 @@
             </div>
 
             <div class="page-item" style="background-color: green; min-height: 350px">
-                <h2>Map</h2>
+                <svg id="map" viewBox="0 0 650 400" preserveAspectRatio="xMaxYMax meet">
```

Comment:
Not sure, but to improve loading times we want to load the smallest number of files possible.
So we could do it, and have gulp compile it to a single `html` file.